server/protocols/jmap-integration-testing/src/test/java/org/apache/james/jmap/methods/GetMailboxesMethodTest.java

```diff
@@ -342,4 +342,25 @@ public void getMailboxesShouldReturnMailboxesWithSortOrder() throws Exception {
                         ImmutableMap.of("name", "inbox", "sortOrder", 10));
     }
 
+    @Test
+    public void getMailboxesShouldReturnMailboxesWithRolesInLowerCase() throws Exception {
+        jmapServer.serverProbe().createMailbox(MailboxConstants.USER_NAMESPACE, username, "outbox");
+
+        String response = given()
+            .accept(ContentType.JSON)
+            .contentType(ContentType.JSON)
+            .header("Authorization", accessToken.serialize())
+            .body("[[\"getMailboxes\", {}, \"#0\"]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .content(startsWith("[[\"mailboxes\","))
```

Comment:
use of "inline" json assertions (using .body()) would be more readable and less fragile (startsWith, json formatters and white spaces, risky bet!).